package.json

```diff
@@ -91,7 +88,10 @@
     "**/ember-electron/**/minimatch": "^3.0.5",
     "**/babel-core/json5": "^2.2.2",
     "**/react-syntax-highlighter/refractor/prismjs": "^1.27.0",
-    "**/@storybook/**/immer": "^9.0.6"
+    "**/@storybook/**/immer": "^9.0.6",
+    "**/nomnom/underscore": "^1.12.1",
+    "**/normalize-package-data/hosted-git-info": "^3.0.8",
+    "**/remark-parse/trim": "~1.0.1"
```

Comment:
It doesn't matter too much but just curious does it actually need to be `~` vs `^`?

Reply:
@ZedLi 
I just copied what was used before to resolve `trim`. But you are right, in this case I would say `^` is better, because it includes for minor patches to be resolve, while `~` will just allow patches. I will update the PR.
Nice catch!! and appreciate the feedback :wink: